lesson_13/gallery.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Magra:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/iconfont.css">
+    <link rel="stylesheet" href="styles/reset.css">
+    <link rel="stylesheet" href="styles/general.css">
+    <link rel="stylesheet" href="styles/gallery.css">
+    <title>CSS Layouts</title>
+</head>
+<body>
+    <div class="wrapper">
+
+        <!-- Header -->
+        <header class="header">
+            <div class="header__worktime">
+                <p>Mo - Fr: 8:00 - 17:00</p>
+                <p>other days on request</p>
+            </div>
+            <div class="header__logo">
+                <a href="#"><img src="img/logo.png" alt=""></a>
```

Comment:
could be link to index.html
and why alt attribute is empty?

Reply:
Ok!